taprpc/mintrpc/mint.proto

```diff
@@ -187,6 +187,20 @@ message MintAsset {
     a BIP-86 key will be derived from the underlying wallet.
     */
     taprpc.ScriptKey script_key = 12;
+
+    /*
+    DecimalDisplay dictates the number of decimal places to shift the amount to
+    the left converting from Taproot Asset integer representation to a
+    UX-recognizable fractional quantity.
+
+    For example, if the decimal_display value is 2 and there's 100 of those
+    assets a wallet would display the amount as "1.00". This field is intended
```

Comment:
s/of those assets a wallet/of those assets and a wallet/

Reply:
hm, not really a syntax that matches what I want to say here, maybe:
```
if the decimal_display value is 2 and there's 100 of those
assets, then a wallet would display the amount as "1.00"
```